go-fuzz-build/main.go

```diff
@@ -259,12 +260,29 @@ func (c *Context) populateWorkdir() {
 	// GOROOT/pkg/tool and GOROOT/pkg/include.
 	// Even better, see if we can avoid making some copies
 	// at all, using some combination of env vars and toolexec.
-	c.copyDir(filepath.Join(c.GOROOT, "pkg", "tool"), filepath.Join(c.workdir, "goroot", "pkg", "tool"))
+
+	sem := make(chan int, 10)
+	var wg sync.WaitGroup
+	do := func(fn func()) {
+		wg.Add(1)
+		sem <- 0
+		fn()
```

Comment:
Wait, shouldn't this include `go` somewhere? ;)

Reply:
Ouch. :P That's what I get for refactoring while benchmarking.

I just re-benchmarked with higher N (with `go`!) and can't reproduce the speedup. I'll force push this branch to remove this commit, at least for now.